src/modules/reducers/web3Connect.js

```diff
@@ -85,6 +85,7 @@ export const setupWeb3 = () => {
                 console.debug(
                     "No web3 detected. Falling back to http://localhost:8545. You should remove this fallback when you deploy live, as it's inherently insecure. Consider switching to Metamask for development. More info here: http://truffleframework.com/tutorials/truffle-and-metamask"
                 );
+                //const web3Provider = new Web3.providers.WebsocketProvider("ws://localhost:8545");
```

Comment:
If we do not need this commented line please delete it.

Reply:
done :)